### PR TITLE
[function](signature) add datev2 signature for from_days function

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes

add datev2 signature for from_days function

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/FromDays.java [hunk 1]
```diff
@@ -38,6 +39,7 @@ public class FromDays extends ScalarFunction
         implements UnaryExpression, ExplicitlyCastableSignature, AlwaysNullable {
 
     public static final List<FunctionSignature> SIGNATURES = ImmutableList.of(
+            FunctionSignature.ret(DateV2Type.INSTANCE).args(IntegerType.INSTANCE),
             FunctionSignature.ret(DateType.INSTANCE).args(IntegerType.INSTANCE)
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/FromDays.java:43]: should not have two signature with same input. remove sig with date return type. and update from_days constant folding in `org.apache.doris.nereids.trees.expressions.functions.executable.DateTimeExtractAndTransform`

**Review by morrySnow — CHANGES_REQUESTED** [review]

**Comment by zhangstar333** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**